### PR TITLE
Initial add of IA Guide

### PR DESCRIPTION
**Pull request by Loquacity**
> __ALL Pull-Requests require an associated [Issue](https://github.com/thegooddocsproject/templates/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) as documented in [How to contribute](https://github.com/thegooddocsproject/templates/blob/master/CONTRIBUTING.md#contributing).__

Partial: https://github.com/thegooddocsproject/planning/issues/16

## Purpose / why

This PR adds the IA Guide (interactive and static) to the website, along with the associated GH action.

## What changes were made?

Adds three new directories:

* `ia-guide`: contains the files for both guides, including associated images, etc.
* `.github`: contains the workflow for the GH action
* `storyformats`: contains the Chapbook format required for building the interactive guide.

It also adds a new page to the nav for the guides, and a short landing page to jump off to the two versions of the IA Guide

## Verification

- [ ] Check that both versions of the IA Guide are building correctly, with all functions and images working as expected
- [ ] Check that the GH Action is publishing the interactive guide correctly

**IMPORTANT** If you have feedback on the content of the IA Guide, please create an issue and I will address them after the guides have landed. Please use this PR for implementation only.

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Are issues linked correctly?
* [ ] Is this PR labeled correctly?
* [ ] ~If template updates: do they align with [developers.google.com/style/](https://developers.google.com/style/)?~
* [ ] Did the PR receive at least one :+1: and no :-1: from core-maintainers?
* [ ] On merging, did you complete the merge using [keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue)?
* [ ] On merging, did you add any applicable notes to a [draft release](https://github.com/thegooddocsproject/templates/releases) and link to this PR?
* [ ] On merging, go back and delete the IA Guide directory from the incubator repo.

### CODE REVIEW

File: ia-guide.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Information Architecture Guide
```
Comment (mgan59) [inline ia-guide.md:5]: Need to add `yaml` to the top

```yaml
---
layout: default
---
```

File: ia-guide/ia-static.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
+# Information Architecture
```
Comment (mgan59) [inline ia-guide/ia-static.md:5]: Need to add `yaml` to the top

```yaml
---
layout: default
---
```

File: ia-guide.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+There are two version of the Information Architecture Guide.
+
+Use the interactive version for an immersive experience, and choose your own adventure through the process.
+
+[Interactive Information Architecture Guide](ia-guide/ia-cyoa.html)
+
+Or, if you just want a quick refresher, check out the static guide instead.
+
+[Static Information Architecture Guide](ia-guide/ia-static.md)
```
Comment (mgan59) [inline ia-guide.md]: change this link to `ia-guide/ia-static.html` and not the `.md`

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+          go get github.com/tmedwards/tweego
+          export PATH=$PATH:$(go env GOPATH)/bin
+          tweego -f chapbook-1 ia-guide/ia-cyoa.tw -o ia-guide/ia-cyoa.html
+
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v2
+        env:
+          PERSONAL_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PUBLISH_BRANCH: gh-pages
```
Comment (mgan59) [inline .github/workflows/build.yml:27]: 👌 this is actually working 🎉

**Review by mgan59 — CHANGES_REQUESTED** [review]
I noted some changes you need to make -- mainly adding `yaml` to define the `layout:default` for the makrdown files.  Otherwise jekyll will not build the file as an html asset.  Otherwise, the IA-interactive stuff is building -- great work 👍

File: .github/workflows/build.yml (hunk 4 above)
Comment (Loquacity) [reply .github/workflows/build.yml:27]: Hooray! :tada:

File: ia-guide/ia-static.md [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+It has a polar bear on the cover.
+
+![A reclining smiling banana with red writing saying "lets go banana](images/banana.svg "Lets go banana")
+
+## That's it!
+
+You've reached the end of the static Information Architecture Guide.
+If you're ready to get started, go [check out the templates](https://github.com/thegooddocsproject/templates/blob/master/README.md)
+
+If you want to try the interactive version of this guide, visit the [interactive IA Guide](don't click here the link does not exist yet).
```
Comment (mgan59) [inline ia-guide/ia-static.md]: @Loquacity This is the last link you need to update -- `ia-guide/ia-cyoa.html`

**Review by mgan59 — APPROVED** [review]
Pulled down your changes and ran through the site, your updated `yaml` in the pages is working as expected.  I noted the one remaining link that needs an update.

File: ia-guide/ia-static.md (hunk 5 above)
Comment (Loquacity) [reply ia-guide/ia-static.md]: Oh, I missed one! Thanks :+1:

**Review by jaredmorgs — APPROVED** [review]
This is ready to merge in.

**Pull request merged**